### ISSUE TITLE
Error when starting ASF with ASFEnhance

### ISSUE THREAD

**Issue by Botan626** [chr233/ASFEnhance#129]
**Describe the bug / Bug 描述**

```
2023-07-13 00:29:44|ArchiSteamFarm-1|ERROR|ASF|Register checkout domain failed() System.ArgumentException: An item with the same key has already been added. Key: https://checkout.steampowered.com/
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey, TValue, InsertionBehavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at ASFEnhance.ReflectionHelper.AddService()
   at ASFEnhance.ASFEnhance.<>c.<<OnASFInit>b__14_0>d.MoveNext()
```

ASF Version  5.4.8.0
ASFEnhance Version 1.8.6.2
"DevFeature": true

**Comment by chr233** [comment]
this version of ASFEnhance depends on ASF 5.4.7.3, ASF 5.4.8.0 is not tested, and maybe not incompatible

**Comment by Botan626** [comment]
ASF v5.4.8.3 is stable now and this error is there.

**Comment by Botan626** [comment]
fixed in v1.8.8.0, thanks
